scripts/_common.sh

```diff
@@ -1,5 +1,23 @@
 #!/bin/bash
 
+#=================================================
+# COMMON VARIABLES
+#=================================================
+
+# dependencies used by the app
+pkg_dependencies="apt-transport-https"
+
+nodejs_version="10"
+
+#=================================================
+# PERSONAL HELPERS
+#=================================================
+
+#=================================================
+# EXPERIMENTAL HELPERS
+#=================================================
```

Comment:
Note that these helpers are probably now official in 3.8 ?

Reply:
Those helpers have been removed